Automattic/pocket-casts-ios
# [Winback] Plans view

| 📘 Part of: #2435 
|:---:|

Fixes #2443 

| Available Plans |
| -------- |
| ![RocketSim_Screenshot_iPhone_16_6 1_2024-12-04_17 25 02](https://github.com/user-attachments/assets/5bb328da-24af-4dad-8334-4d99e0cc92e7) |

## To test

- CI must be 🟢 
- Make sure you enabled `winback` FF
- Use a real device with a sandbox account
- Use a Plus/Patron account or a free account and purchase a Plan
- Got to your account, Cancel Subscription
- Tap on `Looking for a different Plan`
- ✅ Confirm you should see the list of plan. Your current plan should be selected
- Tapping on the same plan shouldn't do anything
- Tap on a different plan you should see the IAP panel prompted
- You can try to purchase and change plan with a sandbox account

**Important:**
The rows with Indaco theme are not properly displayed. We will address this in a new PR.

## Checklist
